classes/cache/Cache.php

```diff
@@ -181,7 +181,9 @@ public static function getInstance()
     {
         if (!self::$instance) {
             $caching_system = _PS_CACHING_SYSTEM_;
-            self::$instance = new $caching_system();
+            if (class_exists($caching_system)) {
```

Comment:
I remember some issues related to the nonexisting caching class, nice

Reply:
:+1: